tests/php/Jobs/CloneImagesOrVideosTest.php

```diff
@@ -45,7 +47,9 @@ public function testCloneImageVolume()
             'volume_id' => $volume->id,
```

Comment:
Please add a newline after the `=`, indent the array elements and add a trailing comma.